packages/reakit/src/Button/__examples__/ButtonWithTootltip/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import { Button } from "reakit/Button";
+import { Tooltip, TooltipReference, useTooltipState } from "reakit/Tooltip";
+import { VisuallyHidden } from "reakit/VisuallyHidden";
+
+type Props = {
+  isLocked: boolean;
+  onClick: () => void;
+};
+
+function LockButton({ isLocked, ...rest }: Props) {
+  const tooltip = useTooltipState();
+  return (
+    <>
+      <TooltipReference {...rest} {...tooltip} as={Button}>
+        {isLocked ? (
+          <>
+            <VisuallyHidden>Click to unlock</VisuallyHidden>{" "}
+            <span aria-hidden>🔒</span>
+          </>
+        ) : (
+          <>
+            <VisuallyHidden>Click to lock</VisuallyHidden>{" "}
+            <span aria-hidden>🔓</span>
+          </>
+        )}
```

Comment:
Nice usage of `VisuallyHidden` and `<span aria-hidden>` 👍

Could we make it more simple? Maybe something like this (haven't tested it)?
```suggestion
        <>
          <VisuallyHidden>Click to {isLocked ? "unlock" : "lock"}</VisuallyHidden>{" "}
          <span aria-hidden>{isLocked ? "🔒" : "🔓"}</span>
        </>
```

We can also abstract these labels into variables outside the JSX.